react-view-model.js

```diff
@@ -55,3 +58,9 @@ module.exports = namespace.reactViewModel = function reactViewModel(displayName,
 
 	return App;
 };
+
+// Expose Component and helpers to named imports
+module.exports.Component = Component;
+module.exports.promise = promise;
+module.exports.autoBindMethods = autoBindMethods;
```

Comment:
should `autoBindMethods` be `autobindMethods` to match the filename?

Reply:
Good question. I'll change it one way or the other, but which is correct?

Is it `auto-bind-methods` and `autoBindMethods`?
or `autobind-methods` and `autobindMethods`?

Reply:
i like autobind-methods. If you did it with spaces, you'd do "autobind methods", not "auto bind methods", as "auto" is not its own word.